### PR TITLE
always cleanup the rails db

### PR DESCRIPTION
**Pull request by MaximilianMeister**
and use rake for cleaning up

### CODE REVIEW

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -278,6 +278,16 @@ post_fail_handler ()
     rm -f $crowbar_install_dir/crowbar_installing
 }
 
+function rails_db_cleanup()
+{
+    rm -f $crowbar_install_dir/crowbar-install-failed
+    rm -f $installation_steps
+    pushd .
```
Comment (tboerger) [inline scripts/install-chef-suse.sh]:
```
pushd /opt/dell/crowbar_framework > /dev/null
rake db:migrate:reset
popd  > /dev/null
```

**Comment by tboerger** [comment]
A tiny improvement, afterwards you get my :+1:

File: scripts/install-chef-suse.sh [hunk 2]
```diff
@@ -278,6 +278,15 @@ post_fail_handler ()
     rm -f $crowbar_install_dir/crowbar_installing
 }
 
+function rails_db_cleanup()
+{
+    rm -f $crowbar_install_dir/crowbar-install-failed
+    rm -f $installation_steps
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: So these two bits are not really related to db cleanup, so maybe rename the function (reset_crowbar, maybe -- not super good name either, though)?

File: scripts/install-chef-suse.sh [hunk 3]
```diff
@@ -278,6 +278,15 @@ post_fail_handler ()
...
+    pushd /opt/dell/crowbar_framework > /dev/null
+    rake db:migrate:reset
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: Nice, thanks for doing that! But doesn't that remove the existing session from the db; is it an issue if installing from the webui?

Comment (mjura) [inline scripts/install-chef-suse.sh]: Yeap, I worry about dropping whole database and recreating it once again

Comment (tboerger) [inline scripts/install-chef-suse.sh]: As mentioned in the previous pull request we can migrate down to `sessions` table to be keep the session data working. An example will follow.

Comment (tboerger) [inline scripts/install-chef-suse.sh]: https://github.com/crowbar/crowbar-core/pull/141

Comment (tboerger) [inline scripts/install-chef-suse.sh]: You can call `bin/rake db:cleanup` with the mentioned pull request.

**Comment by MaximilianMeister** [comment]
depends on https://github.com/crowbar/crowbar-core/pull/141

**Comment by rsalevsky** [comment]
:+1:

**Comment by tboerger** [comment]
Same functionality as before but with cleaner handling, merging.
